pyscriptjs/esbuild.mjs

```diff
@@ -0,0 +1,72 @@
+import { build } from 'esbuild';
+import { spawn } from 'node:child_process';
+import { watch } from 'node:fs';
+import { cp } from 'node:fs/promises';
+
+const production = !process.env.NODE_WATCH || process.env.NODE_ENV === 'production';
+
+const copy_targets = [
+    { src: 'public/index.html', dest: 'build' },
```

Comment:
I don't have opinions here, I've just brought over what we were copying before. Happy to drop unnecessary stuff ... but to some extend, as we removed 6 seconds from the equation, I wouldn't personally care much if an extra file is moved over a destination folder because that's not where time bundling was spent before.

Reply:
I am flagging this as resolved, if that's OK, as it's not related to this PR.

Reply:
I've opened an issue about this https://github.com/pyscript/pyscript/issues/1302